code/modules/organs/external/external.dm

```diff
@@ -304,132 +325,162 @@
 			   DAMAGE PROCS
 ****************************************************/
 
-/obj/item/organ/external/proc/is_damageable(var/additional_damage = 0)
-	//Continued damage to vital organs can kill you, and robot organs don't count towards total damage so no need to cap them.
-	return (vital || (robotic >= ORGAN_ROBOT) || brute_dam + burn_dam + additional_damage < max_damage)
+/obj/item/organ/external/proc/is_damageable(check_damage_cap)
+	// todo: rework
+	if(check_damage_cap)
+		//Continued damage to vital organs can kill you, and robot organs don't count towards total damage so no need to cap them.
+		// todo: this is absolutely fucking stupid, rework asap.
+		if(vital || (robotic >= ORGAN_ROBOT) || brute_dam + burn_dam < max_damage)
+			return TRUE
+		return FALSE
+	return TRUE
```

Comment:
Might as well guard clause or just do a leading . = FALSE

Reply:
it's awful anyways tbh, it'll be rewritten in med update